scripts/add_extra_components.py

```diff
@@ -70,9 +70,11 @@
 
 
 def attach_storageunits(n, costs, config):
-    elec_opts = config["electricity"]
-    carriers = elec_opts["extendable_carriers"]["StorageUnit"]
-    max_hours = elec_opts["max_hours"]
+
+    carriers = config["electricity"]["extendable_carriers"]["StorageUnit"]
+    carriers_classic = [x for x in carriers if x == "H2" or x == "battery"]
```

Comment:
ok for list comprehension by maybe using `... if x not in / in ["H2", "battery"]`

Alternative solution may be something like
``` python
implemented_classic_carriers = set(["H2", "battery"])
carriers_classic = implemented_classic_carriers.difference(set(carriers))
carriers_database = set(carriers).difference(implemented_classic_carriers)
```